pom.xml

```diff
@@ -283,17 +283,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>mrm-maven-plugin</artifactId>
-        <version>1.5.0</version>
-        <configuration>
-          <propertyName>repository.proxy.url</propertyName>
```

Comment:
The `repository.proxy.url` a few lines above near line 250 can also be deleted now.